packages/remix-node/rollup.config.js

```diff
@@ -24,7 +24,7 @@ module.exports = function rollup() {
   return [
     {
       external: (id) => isBareModuleId(id),
-      input: `${SOURCE_DIR}/index.ts`,
+      input: [`${SOURCE_DIR}/index.ts`, `${SOURCE_DIR}/install.ts`],
```

Comment:
Now that we have the exports field, we can point to the `dist` folder and remove the hard-coded `install.js` and `install.d.ts` files.